packages/@uppy/status-bar/src/calculateProcessingProgress.ts

```diff
@@ -22,5 +27,5 @@ export default function calculateProcessingProgress (files) {
     mode,
     message,
     value,
-  }
+  } as FileProcessingInfo
```

Comment:
actually in this case because `calculateProcessingProgress ` has a declared a return type `FileProcessingInfo`, we don't need to make an intermediary at all, and we can simply remove `as`.

```suggestion
  }
```

Reply:
```
packages/@uppy/status-bar/src/calculateProcessingProgress.ts:26:3 - error TS2322: Type '{ mode: "indeterminate" | "determinate"; message: string | undefined; value: number; }' is not assignable to type 'FileProcessingInfo'.
  Types of property 'mode' are incompatible.
    Type '"indeterminate" | "determinate"' is not assignable to type '"determinate"'.
      Type '"indeterminate"' is not assignable to type '"determinate"'.

26   return {
```